src/model/sample.rs

```diff
@@ -47,7 +47,8 @@ pub fn prob_read_snv(record: &bam::Record, cigar: &[Cigar], start: u32, variant:
         for c in cigar {
             match c {
                 // potential SNV evidence
-                &Cigar::Match(l) | &Cigar::Diff(l) if contains_start(pos, l) => {
+                &Cigar::Match(l) | &Cigar::Diff(l) | &Cigar::Equal(l) if contains_start(pos, l) => {
+                    qpos = start - pos as u32;
```

Comment:
Well, my simulated ProSolo testing data gave ... unexpected results. The observations flag and the debug output were of great help!